packages/mendel-config/src/variation-config.js

```diff
@@ -10,10 +10,10 @@ const validate = createValidator({
 function VariationConfig(config) {
     const variations = parseVariations(config);
     const allVariationDirs = getAllDirs(variations);
-
     const baseVariation = {
         id: config.baseConfig.id,
         chain: [config.baseConfig.dir],
+        dir: config.baseConfig.dir
```

Comment:
I like this!

Thinking out loud: I think in the future we should do this for every variation and refactor `variationMatches` to have this instead. I am thinking about it too. Should we validate further making sure we have exactly one canonical dir per variation, even if we don't match the IDs? It is probably a nice idea.